src/language-js/printer-estree.js

```diff
@@ -4087,15 +4087,45 @@ function printFunctionTypeParameters(path, options, print) {
 
 function printFunctionParams(path, print, options, expandArg, printTypeParams) {
   const fun = path.getValue();
+  const parent = path.getParentNode();
   const paramsField = fun.parameters ? "parameters" : "params";
+  const isParametersInTestCall = isTestCall(parent);
+  const isParametersShouldBeHugged = shouldHugArguments(fun);
+  const isParametersShouldBeExpanded =
```

Comment:
`isParametersShouldBeHugged` -> `shouldHugParameters` ?

Reply:
/cc @duailibe done, also rename `isParametersShouldBeExpanded` to `shouldExpandParaments`